inertia/components/authentification-bloc.tsx

```diff
@@ -0,0 +1,121 @@
+import { useState } from 'react'
+import { Link, router, usePage } from '@inertiajs/react'
+import { Alert } from '@codegouvfr/react-dsfr/Alert'
+import { Button } from '@codegouvfr/react-dsfr/Button'
+import { Checkbox } from '@codegouvfr/react-dsfr/Checkbox'
+import { Input } from '@codegouvfr/react-dsfr/Input'
+import { fr } from '@codegouvfr/react-dsfr'
+import Avatar from '@codegouvfr/react-dsfr/picto/Avatar'
+
+export default function AuthentificationBloc() {
+  const borderColor = fr.colors.decisions.border.default.grey.default
+  const linkColor = fr.colors.decisions.text.actionHigh.blueFrance.default
+
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [rememberMe, setRememberMe] = useState(false)
+  const [emailError, setEmailError] = useState('')
+
+  const { errors } = usePage().props
+
+  function handleSubmit(e: { preventDefault: () => void }) {
+    e.preventDefault()
+
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
+
+    if (!emailRegex.test(email)) {
+      setEmailError('Veuillez saisir une adresse e-mail valide')
+      return
+    }
+
+    setEmailError('')
+
+    router.post('/login', {
+      email,
+      password,
+      remember_me: rememberMe,
+    })
+  }
+
+  return (
+    <div className="flex justify-center items-center w-full">
+      <div
+        className="flex flex-col justify-center items-center fr-p-3w w-full max-w-[700px]"
+        style={{ border: `1px solid ${borderColor}`, borderRadius: '8px' }}
+      >
+        <div className="fr-m-4w">
```

Comment:
```suggestion
        <div>
```

Je pense qu'il vaut mieux retirer le margin. Ça fait un espace trop important entre l'entête et les champs et ça augmente de trop l'espace au niveau de la bordure supérieure